config.js

```diff
@@ -35,6 +36,10 @@ const DEFAULT_DB_CONNECTION_POOL = { // sequelize defaults
   idle: 10000,
 };
 
+// Enable Rooms Routes
+const swagger = featureToggles.isFeatureEnabled('enableRooms') ?
+  './api/v1/rooms-swagger.yaml' : './api/v1/swagger.yaml';
```

Comment:
So now we have to make any swagger changes in *both* files? There must be a better way, e.g. keep in two separate files and if feature is enabled, load from both files?